A6httplib/A6httplib.h

```diff
@@ -0,0 +1,31 @@
+#ifndef A6httplib_h
+#define A6httplib_h
+
+#include <Arduino.h>
+//#include <A6lib.h>
+
+class A6httplib
+{
+private:
+    String _host;
+    int _port;
+    String _path;
+    String _apn;
+    A6lib *_A6l;
+
+public:
+    A6httplib();
+    A6httplib(A6lib *);
+
+    ~A6httplib();
+
+    bool ConnectGPRS(String apn);
+
+    bool AddHeader(String header);
```

Comment:
Oh alright, let me know when it's done! Also, could you run `astyle` on the code, for consistency? I use this config file/options:

~~~
style=java
unpad-paren
pad-header
add-brackets
convert-tabs
pad-oper
~~~